src/Http/BaseApplication.php

```diff
@@ -110,6 +118,10 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
      */
     protected function getDispatcher()
     {
-        return new ActionDispatcher(null, null, DispatcherFactory::filters());
+        if ($this->dispatcher === null) {
```

Comment:
This method is called while the class is being invoked. The dispatcher wasn't created in a constructor before. What's different now?